crates/serpen/tests/fixtures/comprehensive_ast_rewrite/main.py

```
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+"""
+Comprehensive AST rewriter test fixture - Main entry point
+This module demonstrates complex naming conflicts and import scenarios
+"""
+
+# Import conflicts: 'process' function exists in multiple modules
+from core.database.connection import process as db_process
+from core.utils.helpers import process, Logger as UtilLogger, validate
+from services.auth.manager import process as auth_process, User, validate as auth_validate
+from models.user import User as UserModel, process_user, Logger
```

Ruff (0.11.9)

`models.user.process_user` imported but unused

Remove unused import: `models.user.process_user`

(F401)